src/price-formatting/parse.js

```diff
@@ -96,84 +96,99 @@ function parseDecimalprice(numberFormatting, s, formatFlags) {
     return 0;
 }
 
-function parseFractionalPrice(numberFormatting, s, formatFlags, decimals) {
-
+function parseModernFractionalPrice(numberFormatting, s, decimals) {
     let result;
+    const separator = getModernFractionsSeparator(numberFormatting);
+    const denominator = 1 << decimals;
+
+    const pipIndex = s.indexOf(separator);
+    if (pipIndex !== -1) {
+        const integerPart = s.substring(0, pipIndex).trim();
+        let isNegative = false;
+        if (integerPart.length > 0) {
+            result = numberFormatting.parse(integerPart);

```

Comment:
see number parse and I'd suggest doing this on the whole string e.g.

const signInfo = parseNumberNegativePattern(value, options, true);
    let sign = signInfo[0];
    const num = signInfo[1];
    if (sign === '') {
        sign = '+';
    }

this matches what we do in formatPriceParts where we add negative pre and negative post